features/step_definitions/networking.rb

```diff
@@ -683,3 +683,15 @@
   cb[cb_name] = @result[:response].chomp
   logger.info "The node's default interface is stored in the #{cb_name} clipboard."
 end
+
+Given /^I restart the ovs pod on the#{OPT_QUOTED} node$/ do | node_name |
+  ensure_admin_tagged
```

Comment:
if this restart can affect other users of the cluster running stuff at the same time, then this needs to be `ensure_destructive_tagged`.

Reply:
updated, thanks

Reply:
/lgtm. yep, it seems to be an ensure_destructive_tagged